com.expidusos.file_manager.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.expidusos.file_manager
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: file_manager
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=system-bus
+  - --socket=session-bus
```

Comment:
these broad dbus permission are not allowed. You should specify the dbus name you need instead.

Reply:
Alright, I've limited the DBus to what I only need.